cli/tests/unit/jupyter_test.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+import { assertThrows } from "./test_util.ts";
+
+Deno.test("Deno.jupyter is not available", () => {
+  assertThrows(
+    () => Deno.jupyter,
+    "Deno.jupyter is only available in `deno jupyter` subcommand.",
+  );
+});
```

Comment:
How should we test the actual functionality under `Deno.jupyter`? Do we have to somehow monkeypatch to allow it then test it?

Reply:
I added another cell in `integration_test.ipynb` notebook - if you run it manually there should be no changes in Git

Reply:
Ahhhh beautiful